harbor/tests/conftest.py

```diff
@@ -50,6 +50,7 @@ def dd_environment(e2e_instance):
     conditions = [
         CheckDockerLogs(compose_file, expected_log, wait=3),
         lambda: time.sleep(4),
+        CheckEndpoints(URL + USERS_URL),
```

Comment:
Not for this PR, but `USERS_URL` should be `USERS_ENDPOINT` or `USERS_PATH`. I was confused why you were concatenating two urls, but then I saw that one of them isn't a url at all :)

Reply:
nah you're right, I was also confused and got lazy. I updated it and also updated the condition to have a better one